src/components/ReportActionItemIOUAction.js

```diff
@@ -16,28 +17,38 @@ const propTypes = {
     /** The associated chatReport */
     chatReportID: PropTypes.number.isRequired,
 
-    /** Should render the preview Component? */
-    shouldDisplayPreview: PropTypes.bool.isRequired,
+    /** Is this IOUACTION the most recent? */
+    isMostRecentIOUReportAction: PropTypes.bool.isRequired,
 
     /* Onyx Props */
     /** ChatReport associated with iouReport */
     chatReport: PropTypes.shape({
         /** The participants of this report */
         participants: PropTypes.arrayOf(PropTypes.string),
     }),
+
+    iouReport: PropTypes.shape({
+        /** Does the iouReport have an outstanding IOU? */
+        hasOutstandingIOU: PropTypes.bool,
```

Comment:
I would suggest rather than passing three different pieces of data as props to control one thing, to just pass a single prop that is `showPreview` and let the parent sort out the proper logic.

Reply:
In fact, I see that was there before and I'm not sure why it was removed because that made it much more simple.

Sorry if I missed that discussion somewhere!

Reply:
Marking resolved, after a discussion.